### ISSUE TITLE
[Bug]: View not work in Ubuntu 24.04

### ISSUE THREAD

**Issue by shankhadevpadam** [laravel/vs-code-extension#106]
### Extension Version

0.1.14

### PHP Binary

Valet

### Operating System

Linux

### What happened?

![Screenshot from 2024-12-20 20-14-55](https://github.com/user-attachments/assets/43dc14b7-3c0e-4f06-8db7-7512499ccbca)

Unable to navigate view file.

**Comment by mikebronner** [comment]
@shankhadevpadam have you tried putting the view in double quotes? I've seen this helps with other goto extensions that I have used in the past. Also, if you hover over the view file, what is the warning it gives (it has orange squiggly underline)? If the view doesn't exist in that specific location, it may also not be able to navigate to it.

The following works for me where the view file is located at `/resources/views/my/nested/folder/index.blade.php`:

```php
    public function __invoke($logType)
    {
        return view('my.nested.folder.index', compact('logType'));
    }
```

**Comment by shankhadevpadam** [comment]
> @shankhadevpadam have you tried putting the view in double quotes? I've seen this helps with other goto extensions that I have used in the past. Also, if you hover over the view file, what is the warning it gives (it has orange squiggly underline)? If the view doesn't exist in that specific location, it may also not be able to navigate to it.
> 
> The following works for me where the view file is located at `/resources/views/my/nested/folder/index.blade.php`:
> 
> ```
>     public function __invoke($logType)
>     {
>         return view('my.nested.folder.index', compact('logType'));
>     }
> ```


It is not works for me.

**Comment by joetannenbaum** [comment]
Just to confirm, you definitely have a view at `resources/views/users/index.blade.php`?

**Comment by shankhadevpadam** [comment]
Yes definitely. I have view file in same location. But I used https://marketplace.visualstudio.com/items?itemName=codingyu.laravel-goto-view extension for work.

**Comment by TitasGailius** [comment]
Thanks for reporting this. We’re clearing out some older issues that may have been fixed by the latest improvements in the extension. Feel free to reopen if you’re still experiencing anything similar.
